### DISCUSSION TITLE
Add Segmind backend to Visualizer 为 Visualizer 添加 Segmind backend

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#288] [MMEngine]
# Add Segmind backend to Visualizer 
 为 Visualizer 添加 Segmind backend 

Implementing the Segmind backend, supplementing it with unit tests, and updating user documentation. 
 
 Related links:[[https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/logger/segmind.py](https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/logger/segmind.py)](https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/logger/segmind.py](https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/logger/segmind.py)) 

实现 Segmind backend，补充并通过单元测试，更新用户文档 
 
 相关链接：[[https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/logger/segmind.py](https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/logger/segmind.py)](https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/logger/segmind.py](https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/logger/segmind.py)) 

## Task Information

**Repository**：MMEngine

**Difficulty**：Easy

**Points**：10

**Estimated Time Consumption**：7 days

## Task Status

**Claimant's Github ID**：wangerlie

**Claim Time**：2023年08月07日

**PR Link**：https://github.com/open-mmlab/mmengine/pull/1311

## Points to Note

❗️请注意：此任务属于超级视客营任务，不能被领取！请登陆 [超级视客营平台](https://openmmlab.com/activity/codecamp) 按照流程申请任务，我们会尽快审核～

**Comment by wangerlie** [comment]
https://github.com/open-mmlab/mmengine/pull/1311
